projects/angular/src/wizard/API.md

```diff
@@ -0,0 +1,36 @@
+# Wizard-clarity - Style API
+
+| CSS Variable Name          | Description                                                        |
+| -------------------------- | ------------------------------------------------------------------ |
+| --clr-wizard-main-bgcolor | Changes the background color in wizard element.
+| --clr-wizard-stepnav-bgcolor | Changes the background color of the side navigation bar for all wizard elements..
+| --clr-wizard-main-text-color | Changes the color of text in form validation for successfully filling information; in alt net, changes the color of text for checkbox text and in step navigation changes the text color of required information to proceed.
+| --clr-wizard-stepnav-text | Changes the color of text inside the side navigation bar of wizard element for visited items.
+| --clr-wizard-stepnav-text--active | Changes the text's color for the currently active element inside the side nav bar of wizard element.
+| --clr-wizard-title-text | Changes the color of the text in the title of a wizard element. This change affects all wizard elements.
```

Comment:
Drop "The change affects all wizard elements"

Reply:
Updated ,As suggested